### PR TITLE
Create Comment-OriginAlertURL

### PR DESCRIPTION
**Pull request by jross1012**
[Comment-OriginAlertURL.zip](https://github.com/Azure/Azure-Sentinel/files/4512940/Comment-OriginAlertURL.zip)

This playbook will look through the Security Alerts table in the Sentinel enabled workspace. For every event that contains an MCAS Alert, the logic app will post the MCAS Alert URL as a comment in the relevant Sentinel Incident.

By using this playbook, users that have Sentinel and MCAS will be able to better unify their investigation experience by easily being able to navigate to MCAS Alert Activity logs based off of a Sentinel Incident.

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Playbooks/Comment-MCASAlertURL [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            "type": "string"
+        },
+        "UserName": {
+            "defaultValue": "<username>@<domain>",
+            "type": "string"
+        }
+    },
+    "variables": {
+        "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]",
+        "MCASConnectionName": "[concat('mcas-', parameters('PlaybookName'))]",
```
Comment (jross1012) [inline Playbooks/Comment-MCASAlertURL:16]: The MCASConnectionName should be removed. the attached file has this removed. MCAS is not a required connection for the Logic App to run successfully. To receive value from the Logic App, MCAS should be a data connection for Sentinel.

**Comment by dicolanl** [comment]
@jross1012 cool playbook
1. Please add a readme file to the pull request like other playbooks
2.  this playbook could also work for AATP since AATP and MCAS share the same urls.  I think MDATP also has url.  ASC does not and AAD does not.

File: Playbooks/Comment-MCASAlertURL [hunk 2]
```diff
@@ -0,0 +1,283 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "Comment-OriginAlertURL",
```
Comment (jross1012) [inline Playbooks/Comment-MCASAlertURL:6]: Updated name to reflect MCAS, AATP, and MDATP

File: Playbooks/Comment-MCASAlertURL [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+                                "Insert_SubscriptionId": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable"
+                        },
+                        "Run_query_and_list_results": {
+                            "description": "Get all Security Alerts from Sentinel that came from a MCAS, AATP, or MDATP alert",
+                            "inputs": {
+                                "body": "SecurityAlert | where ProviderName == \"MCAS\" or ProviderName == \"MDATP\" or ProviderName == \"Azure Advanced Threat Protection\"",
```
Comment (jross1012) [inline Playbooks/Comment-MCASAlertURL:222]: Added MDATP and AATP to filter

**Comment by jross1012** [comment]
[Comment-OriginAlertURL.zip](https://github.com/Azure/Azure-Sentinel/files/4512980/Comment-OriginAlertURL.zip)

Updated Zip with Readme

**Comment by jross1012** [comment]
@dicolanl - I updated the folder with the readme file and altered the JSON to cover: MCAS, AATP, and MDATP. Thanks for the feedback!

**Pull request merged**